aws/cloudformation/iam.yml.erb

```diff
@@ -835,6 +835,90 @@ roles.each do |name, config| %>
     Type: AWS::IAM::ServiceLinkedRole
     Properties: 
       AWSServiceName: logger.cloudfront.amazonaws.com
+
+  # ------------------------------------------------------------
+  # Create one AWS IAM User for each type of marketing sites environment that GitHub Actions will create, with
+  # permissions needed to create/update/delete a marketing sites CloudFormation Stack.
+  # ------------------------------------------------------------
+  <%
+    marketing_sites_environment_type_configuration = {
```

Comment:
Further down the PR, I see you differentiate sites/clusters by indicating that the `environment_type` is the suffix of the name which flattens the structure

One suggestion is to go with a multi-dimensional configuration with "site" (or "cluster") being one of the dimensions. Although I think your flattened config might make more sense to start with and is less restrictive.

```
marketing_sites_environment_type_configuration = {
   "hoc": { // This one would not be here for this PR, but eventually added
     dev: {}, test: {}, prod: {}
    },
    "cdo": {
      dev: {}, test: {}, prod: {}
    }
}
```